io/src/main/scala/fs2/io/tls/TLSEngine.scala

```diff
@@ -77,6 +77,7 @@ private[tls] object TLSEngine {
 
       def beginHandshake = Sync[F].delay(engine.beginHandshake())
       def session = Sync[F].delay(engine.getSession())
+      def getApplicationProtocol = Sync[F].delay(engine.getApplicationProtocol())
```

Comment:
Need to add this to `TLSSocket` too

Reply:
Should be there now, changed to be consistent with session.